plugins/nanoleaf/README.md

```diff
@@ -7,10 +7,10 @@ This plugin uses the streaming external control effects functionality of
 the Aurora OpenAPI.
 
 Currently only the [Nanoleaf
-Aurora](https://nanoleaf.me/en/consumer-led-lighting/products/
-smarter-series/nanoleaf-light-panels-smarter-kit/)
+Aurora](https://nanoleaf.me/en/consumer-led-lighting/products/smarter-series/nanoleaf-light-panels-smarter-kit/)
```

Comment:
We can also simplify the URL as they've dropped part of it:
https://nanoleaf.me/en/products/smarter-series/nanoleaf-light-panels-smarter-kit/